scalgrafanalib/core.py

```diff
@@ -139,22 +248,38 @@ def to_json_data(self) -> Json:
 
 @attr.s
 class TimeSeries(core.TimeSeries):
-    """TimeSeries: Allow settings decimals & legend values"""
+    """TimeSeries: Allow settings decimals & legend values and backward-compat for showLegend"""
 
     decimals: int = attr.ib(default=0, validator=attr.validators.instance_of(int))
     legendValues: List[str] = attr.ib(  # pylint: disable=invalid-name
         default=[], validator=attr.validators.instance_of(list)
     )
-    spanNulls: Union[int, bool] = attr.ib(  # pylint: disable=invalid-name
+    spanNulls: Union[int, bool] = attr.ib(
```

Comment:
available in grafanalib (was probably not present before, or missed...) → should be removed